facturacion-web/src/app/landing/menu-panel/menu-panel.component.html

```diff
@@ -1,14 +1,40 @@
 <div class="panel-container">
+  <div class="sidebar-header">
+    <h5>FacturacionWeb</h5>
+  </div>
+  <div class="sidebar-profile">
+    <div class="user-pic">
+      <img class="img-responsive img-rounded" src="assets/images/logo.png" alt="User picture">
+    </div>
+    <div class="user-info">
+      <span class="user-name">{{ userName }}
+      </span>
+      <span class="user-status">
+        <span>Online </span>
+        <img class="img-responsive img-rounded" src="assets/images/online.png" alt="">
+      </span>
+    </div>
+  </div>
+
+  <!--  
   <div class="head-container">
-    <img src="assets/images/logo.png">
+    <img src="assets/images/logo.png" width="50" height="50">
     <h1>{{ userName }}</h1>
   </div>
-
+-->
   <div class="menu-section">
     <div class="item-section">
-      <a class="menu-item">Facturación</a>
-      <a class="menu-item">Facturas emitidas</a>
-      <a class="menu-item">Control de stock</a>
+      <div>
+        <div class="item">
+          <a class="menu-item"><img class="img-responsive img-rounded" src="assets/images/facturacion.png">Facturación</a>
```

Comment:
```suggestion
          <a class="menu-item"><img class="logo-img" src="assets/images/facturacion.png">Facturación</a>
```
para las tres imágenes de los logos

Reply:
se saco el img-responsive img-rounded